src/scss/_main.scss

```diff
@@ -352,31 +373,37 @@
 	@include oGridColspan($span);
 }
 
+@mixin oGridRowMargins($total-cols: $o-grid-columns) {
+	$subgrid-ratio: $o-grid-columns / $total-cols;
+
+	margin-right: auto;
+	margin-left: auto;
+}
+
 /// Base row styles
 ///
 /// @param {String} $grid-mode [$o-grid-mode]
-@mixin oGridRow($grid-mode: $o-grid-mode) {
+@mixin oGridRow($grid-mode: $o-grid-mode, $total-cols: $o-grid-columns) {
 	clear: both;
 	min-width: $o-grid-min-width;
 	// Older browsers get a fixed-width layout
 	max-width: oGridGetMaxWidthForLayout($o-grid-fixed-layout);
-	padding-left: $o-grid-gutter / 2;
-	padding-right: $o-grid-gutter / 2;
 	box-sizing: border-box;
-	margin-left: auto;
-	margin-right: auto;
+
+	@include oGridRowMargins($total-cols);
 
 	// Clearfix
 	zoom: 1;
+	overflow: auto;
```

Comment:
Got it. Unfortunately setting overflow has an undesirable side effect: it sets boundaries on the row and anything inside of it won't be able to bleed out of it. Does it make sense?

The flexbox branch is a proposal that is in beta at the moment. Sorry that happened at the same time as the new grid work, we can rebase later.